keps/sig-cli/4944-kustomize-wasm-plugin/kep.yaml

```diff
@@ -0,0 +1,41 @@
+title: Kustomize Wasm Plugin
+kep-number: 4944
+authors:
+  - "@koba1t"
+owning-sig: sig-cli
+participating-sigs:
+  - sig-cli
+status: implementable
+creation-date: 2024-11-05
+reviewers:
+  - "@varshaprasad96"
+approvers:
+  - "@varshaprasad96"
+
+see-also: []
+replaces:
+  - "/keps/sig-cli/2953-kustomize-plugin-graduation/README.md"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.34"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  beta: TBD
+  stable: TBD
+# # The following PRR answers are required at alpha release
+# # List the feature gate name and the components for which it must be enabled
+# feature-gates:
```

Comment:
Are you planning on gating the access to this functionality through an environment variable, like we do with kubectl or in any other way?